src/core_ocean/mode_forward/mpas_ocn_forward_mode.F

```diff
@@ -627,13 +635,14 @@ function ocn_forward_mode_run(domain) result(ierr)!{{{
                close(22)
             end if
 
-            if(trim(config_sw_absorption_type)=='ohlmann00') call ocn_shortwave_forcing_write_restart(domain)
-
-            if (config_use_ecosysTracers) call ocn_ecosys_forcing_write_restart(domain)
-
-            if (config_use_activeTracers_surface_restoring .and. config_use_surface_salinity_monthly_restoring)  &
-               call ocn_salinity_restoring_forcing_write_restart(domain)
+            ! write forcing restart if any type(s) time varying forcing is used
+            if (trim(config_sw_absorption_type)=='ohlmann00' .or. &
+                config_use_ecosysTracers .or. &
+                (config_use_activeTracers_surface_restoring .and. config_use_surface_salinity_monthly_restoring) .or. &
+                config_use_atmospheric_forcing) then
```

Comment:
@xylar, I like that idea a lot. I'll probably leave it the way it is for now and clean it up in a separate PR. That way this one is focused just on the atmospheric forcing and doesn't touch the other forcings (since I'll have to add that function to each of them). Do you think that makes sense?

Reply:
Yes, I think that would be a nice approach.

Reply:
@sbrus89, can you please make an issue so that we can keep track of this and tag it here before we resolve this issue?